Content.Shared/CCVars.cs

```diff
@@ -102,6 +102,13 @@ public sealed class CCVars : CVars
         public static readonly CVarDef<int> TraitorMaxPicks =
             CVarDef.Create("traitor.max_picks", 20);
 
+        /*
+         * TraitorDeathMatch
+         */
+
+        public static readonly CVarDef<int> TraitorDeathMatchStartingBalance =
+            CVarDef.Create("traitordeathmatch.starting_balance", 20);
```

Comment:
traitordeathmatch might be too long, maybe call it `traitordm` instead?

Reply:
Fixed.